resources/views/dropdown.blade.php

```diff
@@ -4,6 +4,7 @@
     'buttonClassExpanded'    => 'text-theme-primary-500',
     'buttonClass'            => 'text-theme-secondary-400 hover:text-theme-primary-500',
     'dropdownClasses'        => 'w-40',
+    'dropdownOriginClass'    => 'origin-top-right',
```

Comment:
🤔 Where is `dropdownOriginClass` used?

Reply:
@leMaur its used here ArkEcosystem/nodem#142